typegate/tests/runtimes/wasm_wire/wasm_wire.ts

```diff
@@ -45,3 +47,33 @@ export const tg = await typegraph("wasm-wire-ts", (g: any) => {
     ).rename("identity"),
   }, Policy.public());
 });
+
+if (Deno.args.length < 2) {
```

Comment:
Will this section have to be added to each typegraph? How about moving it into a common program that imports the typegraph modules dynamically and does all this stuff? We can even do the same thing for Python.

Reply:
this was a lingering one, already did it in that way. check the `utils/tg_deploy_script.ts/.py`